From d7e9532e73c86d78a3796b36256b316d68895b78 Mon Sep 17 00:00:00 2001
From: Robert Bragg <robert@linux.intel.com>
Date: Tue, 21 May 2013 11:59:36 +0100
Subject: [PATCH] forcibly disable use of libicu

---
 configure.ac | 52 ++++------------------------------------------------
 1 file changed, 4 insertions(+), 48 deletions(-)

diff --git a/configure.ac b/configure.ac
index 588744e..3b5f4ec 100644
--- a/configure.ac
+++ b/configure.ac
@@ -166,57 +166,13 @@ AM_CONDITIONAL(HAVE_CAIRO_FT, $have_cairo_ft)
 
 dnl ==========================================================================
 
-PKG_CHECK_MODULES(ICU, icu-uc, have_icu=true, have_icu=false)
-dnl Fallback to icu-config if ICU pkg-config files could not be found
-if test "$have_icu" != "true"; then
-	AC_CHECK_TOOL(ICU_CONFIG, icu-config, no)
-	AC_MSG_CHECKING([for ICU by using icu-config fallback])
-	if test "$ICU_CONFIG" != "no" && "$ICU_CONFIG" --version >/dev/null; then
-		have_icu=true
-		# We don't use --cflags as this gives us a lot of things that we don't
-		# necessarily want, like debugging and optimization flags
-		# See man (1) icu-config for more info.
-		ICU_CFLAGS=`$ICU_CONFIG --cppflags`
-		ICU_LIBS=`$ICU_CONFIG --ldflags-libsonly`
-		AC_SUBST(ICU_CFLAGS)
-		AC_SUBST(ICU_LIBS)
-		AC_MSG_RESULT([yes])
-	else
-		AC_MSG_RESULT([no])
-	fi
-fi
-
-if $have_icu; then
-	CXXFLAGS="$CXXFLAGS `$PKG_CONFIG --variable=CXXFLAGS icu-uc`"
-	AC_DEFINE(HAVE_ICU, 1, [Have ICU library])
-fi
-AM_CONDITIONAL(HAVE_ICU, $have_icu)
+have_icu=false
+AM_CONDITIONAL(HAVE_ICU, 0)
 
 dnl ==========================================================================
 
-PKG_CHECK_MODULES(ICU_LE, icu-le icu-uc, have_icu_le=true, have_icu_le=false)
-dnl Fallback to icu-config if ICU pkg-config files could not be found
-if test "$have_icu_le" != "true"; then
-	AC_CHECK_TOOL(ICU_CONFIG, icu-config, no)
-	AC_MSG_CHECKING([for ICU_LE by using icu-config fallback])
-	if test "$ICU_CONFIG" != "no" && "$ICU_CONFIG" --version >/dev/null; then
-		have_icu_le=true
-		# We don't use --cflags as this gives us a lot of things that we don't
-		# necessarily want, like debugging and optimization flags
-		# See man (1) icu-config for more info.
-		ICU_LE_CFLAGS=`$ICU_CONFIG --cppflags`
-		ICU_LE_LIBS=`$ICU_CONFIG --ldflags-libsonly --ldflags-layout`
-		AC_SUBST(ICU_LE_CFLAGS)
-		AC_SUBST(ICU_LE_LIBS)
-		AC_MSG_RESULT([yes])
-	else
-		AC_MSG_RESULT([no])
-	fi
-fi
-if $have_icu_le; then
-	AC_DEFINE(HAVE_ICU_LE, 1, [Have ICU Layout Engine library])
-fi
-AM_CONDITIONAL(HAVE_ICU_LE, $have_icu_le)
+have_icu_le=false
+AM_CONDITIONAL(HAVE_ICU_LE, 0)
 
 dnl ==========================================================================
 
-- 
1.8.2.1

