From 09d501af412c0576d84fd8dff1f08385f3397ef1 Mon Sep 17 00:00:00 2001
From: Robert Bragg <robert@linux.intel.com>
Date: Thu, 6 Mar 2014 19:52:06 +0000
Subject: [PATCH] avoid building gtester

---
 glib/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glib/Makefile.am b/glib/Makefile.am
index 1330755..dcb2245 100644
--- a/glib/Makefile.am
+++ b/glib/Makefile.am
@@ -411,7 +411,7 @@ endif
 glib-win32-res.o: glib.rc
 	$(WINDRES) glib.rc $@
 
-bin_PROGRAMS	 = ${INSTALL_PROGS}
+#bin_PROGRAMS	 = ${INSTALL_PROGS}
 
 if OS_UNIX
 
-- 
1.9.0

